runner.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	config2 "github.com/hashicorp/envconsul/config"
```

Comment:
Would you please consider aligning the naming of the imports so the module has the same name everywhere. You used `cfg` above which is fine and wouldn't conflict with the consul-template/config package. Please don't use `config2` (to close to existing `config` for my tastes).